docs/standalone-kubelet-crio.md

```diff
@@ -0,0 +1,139 @@
+Kubelet standalone
+==================
+
+Kubelet may be installed headless, which is without the remaining kubernetes
+cluster components. In that "dummy" mode, kubelet only manages static pods
+locally and cannot schedule a pod onto other nodes.
+
+Example installation
+--------------------
+
+Given a Fedora 28 node already prepared and having a static inventory file for
+it, like:
+```
+$ cat ../hosts
+node ansible_host=<IP for ssh access> ansible_user=fedora
+[k8s-cluster]
+node
+```
+
+The following steps may be done to install headless kubelet with CRI-O and runc
+runtime. No networking plugins will be used, so pods only may be started in the
+host networking mode. Execute these example commands from the clonned kubespray
+repo directory:
+
+* Optional step to configure custom repos
+  (the required packages may be found in the stock updates repository as well):
+```
+$ cat > repos.yml << EOF
+- hosts: node
+  become: yes
+  gather_facts: yes
+  tasks:
+    - yum_repository:
```

Comment:
OpenShift repo is already defined in cri-o roles. Do we need it?

* https://github.com/kubernetes-incubator/kubespray/blob/master/roles/cri-o/tasks/main.yaml#L19

Reply:
well spotted, thanks! fixed